src/state/transactions/updater.tsx

```diff
@@ -118,9 +120,9 @@ export default function Updater(): null {
                   receipt: {
                     blockHash: '0x0',
                     blockNumber: 0,
-                    contractAddress: orderMetadata.settlementContract,
+                    contractAddress: (orderMetadata as any).settlementContract,
```

Comment:
should avoid `any`

Reply:
`settlementContract` is no longer in `orderMetadata`. Good catch!

Reply:
Addressed in [6ed22de](https://github.com/levelkdev/swapr-dapp/pull/1280/commits/6ed22de5368f9372fd2fc6d703d13e4a65f669df)